### ISSUE TITLE
err

### ISSUE THREAD

**Issue by ruslanilyasov** [avocode/avocode#612]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.2
Source UID: 112670
```
Error: incorrect header check
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by vojtatranta** [comment]
This is duplicated issue. More info here: #607

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.

**Comment by ruslanilyasov** [comment]
2016-02-05 23:53, Petr Brzek написав: 

> This error should be now
fixed with the latest Avocode 2.5.3. Please try it and let us know. 
>

> --
> Reply to this email directly or view it on GitHub [1].

Now
everything is fine. Thank you! 

--


----------------------------------------------------------------
Sincerely
yours, Ruslan Ilyasov.
http://ilyasov.eu/
+34 652-489-754



Links:
------
[1]
https://github.com/avocode/avocode/issues/612#issuecomment-180627831
